app/views/split_checkout/_summary.html.haml

```diff
@@ -1,81 +1,95 @@
+            = @order.shipping_method.description
     
-      %a.summary-edit{href: main_app.checkout_step_path(:details)}
-        = t("split_checkout.step3.your_details.edit")
+  %hr
 
-  %div.checkout-substep
-    %div.checkout-title
+  .checkout-substep
+    .checkout-title
       = t("split_checkout.step3.payment_method.title")
-
-    %div.summary
-      %span.summary-value
-        = last_payment_method(@order)&.name
-      %div.summary-description
-        = last_payment_method(@order)&.description
       %a.summary-edit{href: main_app.checkout_step_path(:payment)}
         = t("split_checkout.step3.payment_method.edit")
+    .two-columns
+      %div
+        - payment_method = last_payment_method(@order)
+        = payment_method&.name
+        %em.fees=payment_or_shipping_price(payment_method, @order)
+      %div
+        .summary-subtitle
+          = t("split_checkout.step3.payment_method.instructions")
+        %div
+          = last_payment_method(@order)&.description
 
 
   %div.checkout-substep
     %div.checkout-title
       = t("split_checkout.step3.order.title")
+      %a.summary-edit{href: main_app.cart_path}
+        = t("split_checkout.step3.order.edit")
     
-    = render 'spree/orders/summary', order: @order
+    = render 'spree/orders/summary', order: @order, display_footer: false
+
+
+.summary-right{ "data-controller": "sticky", "data-sticky-target": "container" }
+  .summary-right-line.total
+    .summary-right-line-label= t :order_total_price
+    .summary-right-line-value#order_total= @order.display_total.to_html
+  
+  .summary-right-line
+    .summary-right-line-label= t :order_produce
+    .summary-right-line-value= display_checkout_subtotal(@order)
+
+  - checkout_adjustments_for(@order, exclude: [:line_item]).reverse_each do |adjustment|
+    .summary-right-line
+      .summary-right-line-label= adjustment.label
+      .summary-right-line-value= adjustment.display_amount.to_html
 
-.checkout-step3{"data-controller": "sticky", "data-sticky-target": "container"}
-  - if any_terms_required?(@order.distributor)
-    = render partial: "terms_and_conditions", locals: { f: f }
-  .medium-6
-    .checkout-submit
-      = f.submit t("split_checkout.step3.submit"), name: "confirm_order", class: "button primary", disabled: @terms_and_conditions_accepted == false || @platform_tos_accepted == false
-      %a.button.cancel{href: main_app.checkout_step_path(:payment)}
-        = t("split_checkout.step3.cancel")
+  - if @order.total_tax > 0
+    .summary-right-line
+      .summary-right-line-label= t :order_includes_tax
+      .summary-right-line-value#tax-row= display_checkout_tax_total(@order)
+  
+  .checkout-submit
+    - if any_terms_required?(@order.distributor)
+      = render partial: "terms_and_conditions", locals: { f: f }
+    = f.submit t("split_checkout.step3.submit"), name: "confirm_order", class: "button primary", disabled: @terms_and_conditions_accepted == false || @platform_tos_accepted == false
+   
```

Comment:
There are some weird whitespaces here.

Reply:
Humm ... I don't see what I meant. I don't see any _weird whitespaces_. Sorry, could your tell more about what/where you've found those weird whitespaces?

Reply:
The Github diff looks like this:
![Screenshot from 2023-01-18 15-37-15](https://user-images.githubusercontent.com/3524483/213085263-54a1495e-425b-4d51-a177-70f52ce1df79.png)

I think you have four useless spaces in line 95 and then no newline at the end of the file.

Reply:
Oh yes, good catch! I thought that rubocop was catching this kind of bad formatting.